tiatoolbox/tools/graph.py

```diff
@@ -283,7 +283,7 @@ def build(
             feature_range_thresh (Number):
                 Minimal range for which a feature is considered significant.
                 Features which have a range less than this are ignored.
-                Defaults to 1e-4. If falsey (None, False, 0, etc.), then
+                Defaults to 1e-4. If falsy (None, False, 0, etc.), then
```

Comment:
```suggestion
                Defaults to 1e-4. If False (None, False, 0, etc.), then
```

Reply:
I specifically say falsy not False here because `0` and `None` are not false, but they are 'falsy' i.e. convert to `False` when `__bool__` is used. E.g. `bool(None) == False` even though `None != False`. Cna change the wording if you like. Truthy and falsy are commonly used to describe this behaviour, although they are more commonly used in javascript. See https://docs.python.org/3/library/stdtypes.html#truth and https://developer.mozilla.org/en-US/docs/Glossary/Falsy for more.